kimchi/src/circuits/constraints.rs

```diff
@@ -618,6 +618,10 @@ impl<F: PrimeField + SquareRootField> ConstraintSystem<F> {
     }
 }
 
+//
+pub const NUM_CHUNKS_BY_DEFAULT: usize = 1;
+pub const ZK_ROWS_BY_DEFAULT: u64 = 3;
+
 pub fn zk_rows_strict_lower_bound(num_chunks: usize) -> usize {
```

Comment:
Note: this is *strict*, i.e. it will give 4 when `num_chunks = 2`, and therefore, we should have *at least* 5 in index.cs.zk_rows.
(I am annoying, but could we simply add a small comment please?)